recipes/courtana/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "0.4.3" %}
+{% set name = "courtana" %}
+{% set sha256 = "a8de42d2105f71336eb70a850ca67bff9c8d014c9789e86367bba234099c34c1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py<34]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
```

Comment:
In the source repo, the `requirements.txt` and the `setup.py` put specific restrictions on the versions of the dependencies. Is there a reason to leave them off here?